functions_and_scope_assingnment.js

```diff
@@ -0,0 +1,329 @@
+// // # Function Exercises
```

Comment:
Other than some minor feedback, good job overall with the assignment. Keep coding and keep rocking!